controller/lib/core/sensors.cpp

```diff
@@ -44,14 +44,16 @@ static DebugFloat dbg_volume_uncorrected("volume_uncorrected",
 // Sourced from https://en.wikipedia.org/wiki/Density_of_air
 static const float DENSITY_OF_AIR_KG_PER_CUBIC_METER = 1.225f; // kg/m^3
 
-// Diameters relating to Ethan's Alpha Venturi - II
-// (https://docs.google.com/spreadsheets/d/1G9Kb-ImlluK8MOx-ce2rlHUBnTOtAFQvKjjs1bEhlpM/edit#gid=963553579)
+// Diameters and fudge factor relating to 3/4in Venturi, see
+// https://respiraworks.slack.com/archives/C011ABVBS21/p1591640604065300?thread_ts=1591604076.050000&cid=C011ABVBS21
```

Comment:
Geometry update based on parameters in spreadsheet here: https://docs.google.com/spreadsheets/d/1G9Kb-ImlluK8MOx-ce2rlHUBnTOtAFQvKjjs1bEhlpM/edit#gid=963553579